magda-web-client/src/Components/DistributionIcon.tsx

```diff
@@ -0,0 +1,93 @@
+import defaultFormatIcon from "../assets/format-passive-dark.svg";
```

Comment:
The name and extension makes me expect a React component here - could it be called `IconFunctions.ts` or something like that?